crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -507,6 +544,33 @@ pub(crate) fn read_prettier_files(
         }
     }
 
+    // to be safe, let's drop this content
+    drop(prettier_config_content);
```

Comment:
If you need to do this I’d say it’s a sign you’re doing too much kn a single function.